scripts/build/tools/copy_file.py

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+import argparse
+import os
+import sys
+import glob
+import shutil
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-s', '--src_dir', dest='src_dir', required=True, help='Source directory to copy files from, if not specified, current directory is used.')
+    parser.add_argument('-r', '--file_regex', dest='file_regex', required=True, help='Globbing pattern used to match file names to copy.')
+    parser.add_argument('-t', '--target_dir', dest="target_dir", required=True, help='Target directory to copy files to.')
```

Comment:
Normally command line parameters have dashes to separate words, not underscores.
So I think these parameters shoudl be `--source-dir`, `--file-regex` and `--target-dir`

Reply:
Changed underscore to dash